docs/storage/hotplug_volumes.md

```diff
@@ -85,36 +85,18 @@ $ virtctl addvolume vmi-fedora --volume-name=example-lun-hotplug --disk-type=lun
 ```
 
 ### Retain hotplugged volumes after restart
-In many cases it is desirable to keep hotplugged volumes after a VM restart. It may also be desirable to be able to unplug these volumes after the restart. The `persist` option makes it impossible to unplug the disks after a restart. If you don't specify `persist` the default behaviour is to retain hotplugged volumes as hotplugged volumes after a VM restart. This makes the `persist` flag mostly obsolete unless you want to make a volume permanent on restart.
+In many cases it is desirable to keep hotplugged volumes after a VM restart. It may also be desirable to be able to unplug these volumes after the restart. The `persist` option makes it possible to unplug the disks after a restart. If you don't specify `persist`, the volume is only ever hotplugged on the VMI (instance) level.
 
 ### Persist
-In some cases you want a hotplugged volume to become part of the standard disks after a restart of the VM.
-For instance if you added some permanent storage to the VM. We also assume that the running VMI has a matching VM that defines it specification.
-You can call the addvolume command with the --persist flag. This will update the VM domain disks section in addition to updating the VMI domain disks.
-This means that when you restart the VM, the disk is already defined in the VM, and thus in the new VMI.
+In some cases you want a hotplugged volume to stay hotplugged after a restart of the VM.
+In that case, you can call the addvolume command with the --persist flag.
+This means that when you restart the VM, the disk is already defined in the VM (as hotplugged), and thus is hotplugged to the new VMI.
```

Comment:
I'm a bit confused, isn't this covered in line 88? If so, maybe we can combine the two into one section:
`### Retain hotplugged volumes after restart with `persist` option`

Reply:
+1